getting-started/1_template.md

```diff
@@ -0,0 +1,18 @@
+We’ll start by getting the app up and running, and then explain the different components which we will later extend.
+
+First off, open a terminal and instantiate the template project.
+
+```
+rm -rf create-daml-app
```

Comment:
I think this needs an explanation that you should click on this to copy and execute the command. When I was going through it it wasn’t clear to me if just clicking `continue` is sufficient. Ideally you could only click continue once you’ve run through the steps but I guess katacoda does not support this atm.

Reply:
It does, actually. Let me try out their "step verification" feature